lib/main_micro_alp.pm

```diff
@@ -49,6 +49,7 @@ sub load_config_tests {
 }
 
 sub load_boot_from_disk_tests {
+    loadtest 'boot/boot_to_desktop' if (check_var('MACHINE', 'ppc64le-p10-virtio') || (get_var('WORKER_CLASS') =~ /cpu-power10/ && is_qemu));
```

Comment:
'MACHINE', 'ppc64le-p10-virtio' is WORKER_CLASS cpu-power10 && is_qemu
Why not use one of it ?